plugins/modules/virt_pool.py

```diff
@@ -49,23 +49,15 @@
         type: bool
         description:
             - Specify if a given storage pool should be started automatically on system boot.
-    uri:
-        required: false
-        default: "qemu:///system"
-        description:
-            - I(libvirt) connection uri.
-        type: str
-    xml:
-        required: false
-        description:
-            - XML document used with the define command.
-        type: str
     mode:
         required: false
         choices: [ 'new', 'repair', 'resize', 'no_overwrite', 'overwrite', 'normal', 'zeroed' ]
         description:
             - Pass additional parameters to 'build' or 'delete' commands.
         type: str
+extends_documentation_fragment:
+    - community.libvirt.virt_uri
+    - community.libvirt.virt_xml
 requirements:
     - "python >= 2.6"
     - "python-libvirt"
```

Comment:
This is wrong, it should be libvirt-python

Reply:
we could fix it in a separate PR, thanks for pointing this out